index.rst

```diff
@@ -68,9 +68,11 @@ Visit processing can fail at any point, including within and between tasks, and
 Even within a single worker, the order of events is not fully deterministic (e.g., two snaps could be ingested and processed up to snap combination in parallel, though we have not yet tried to do so).
 However, the possible failures can be divided into a handful of cases:
 
-- Any failure before the end of APDB writing (which is atomic as of October 2023) has no consequences beyond that worker's local repository.
+- Any failure before the start of APDB writing (which is not necessarily atomic) has no consequences beyond that worker's local repository.
```

Comment:
I assume the "not necessarily atomic" refers to APDB writing as a whole, not the "start", which has a reasonably-well-defined temporal location, I hope.